src/config/ConfigBuilder.ts

```diff
@@ -421,8 +460,8 @@ export class ConfigBuilder {
         // Throw in case both memberAdded and memberRemoved are invalid.
         if (typeof membershipListener.memberAdded !== 'function' && typeof membershipListener.memberRemoved !== 'function') {
             throw new RangeError(`Invalid membershipListener is given in 'membershipListeners': ${membershipListener}. `
-                                + 'Expected at least one of \'memberAdded\' and \'memberRemoved\' properties to exist and be a'
-                                + ' function.');
+                + 'Expected at least one of \'memberAdded\' and \'memberRemoved\' properties to exist and be a'
```

Comment:
You could use double quotes in order not to escape the single quotes.

Reply:
We have an eslint rule that forbids double quotes and forces single quotes.

Reply:
Just out of curiosity, why is that rule is used? Why not double quotes? Just for consistency?